README.md

```diff
@@ -15,18 +15,23 @@ møte organises and serves meetings as a drop-in replacement, without needing an
 ### Using møte
 
 Dependencies:
- - Optional: `memcached` (`sudo dnf install memcached`)
- - Python 2.7.x
+ - `memcached` (`sudo dnf install memcached`)
+ - Python 3.8.X
+ - Pipenv
 
 Installation:
- - Optional: create virtual environment
- - `pip install -r requirements.txt`
- - Install using `python setup.py develop`
+ - `pipenv install` will create a new virtual environment and install the requirements. `
```

Comment:
For Pipenv, I usually recommend two steps:

```sh
# Install dependencies
$ pipenv install

# Activate Python virtualenv
$ pipenv shell
```

Reply:
Fixed: https://github.com/fedora-infra/mote/pull/141/commits/8bfdd02674b734fa256248bf2429a00fddf28a11